data/npclib/npc_system/npc_handler.lua

```diff
@@ -610,7 +610,9 @@ if NpcHandler == nil then
 	-- This implements the currently set type of talkdelay.
 	-- The "delay" variable sets the delay for the interval between messages
 	function NpcHandler:say(message, npc, player, delay, textType)
-		if not player then return end
+	if not player then
+		return Spdlog.error("[NpcHandler:say] - Player parameter is missing, nil or not found")
```

Comment:
I think this was removed for some reason, if I remember correctly it seems that when a player logs out with a speech event activated (for example an npc that sends a speech every 1 second), it sends the log to the distro, which messes up the distro without need, since it doesn't matter much if the player ceased to exist, the important thing is to return and not execute the function even if it does not exist.

Reply:
Hmm, faz sentido mesmo.